synapse/handlers/jwt.py

```diff
@@ -109,4 +111,10 @@ def validate_login(self, login_submission: JsonDict) -> str:
         if user is None:
             raise LoginError(403, "Invalid JWT", errcode=Codes.FORBIDDEN)
 
-        return UserID(user, self.hs.hostname).to_string()
+        default_display_name = None
+        if self.jwt_display_name_claim:
+            display_name_claim = claims.get(self.jwt_display_name_claim, None)
```

Comment:
tiny nit - the second parameter to [`dict.get`](https://docs.python.org/3/library/stdtypes.html#dict.get) defaults to `None`:

 ```suggestion
             display_name_claim = claims.get(self.jwt_display_name_claim)
```